.github/workflows/ubuntu.yml

```diff
@@ -13,7 +13,6 @@ env:
 
 jobs:
   pytest-coverage: # from pytest_coverage.yml
-    needs: lint
```

Comment:
@MasterSkepticista can you review this part, as #1193 caused ubuntu workflow failure.

Reply:
Do we need linting for a regular CI stability check? I removed linting check because it is part of every PR merge process. My bad I didn't properly update that block. My suggestion would be to remove lint as a requirement in the periodic workflow. I get that a lint check is not an overhead in CI, but it is best to have all linting checks in one place, for simplicity.

Reply:
P.S.: I am okay either way. I will approve the PR.